charts/backstage/values.yaml

```diff
@@ -308,3 +308,18 @@ serviceAccount:
 
   # -- Auto-mount the service account token in the pod
   automountServiceAccountToken: true
+
+# ServiceMonitor configuration
+serviceMonitor:
+  # -- If enabled, ServiceMonitor resources for Prometheus Operator are created
+  enabled: false
+  # -- Namespace selector for ServiceMonitor resources
```

Comment:
Also, just for house keeping, can you please include an empty line between each property in here? So we keep it consistent with the rest of the chart.